### ISSUE TITLE
City of Tartu: real-time bus data nominally open, but really inaccessible

### ISSUE THREAD

**Issue by tambeta** [okestonia/opendata-issue-tracker#86]
I was happy to find out that Tartu [provides open data](http://www.tartu.ee/et/avaandmed), but the support and accessibility thereof is lacking, to put it mildly. Specifically, I'm interested in the real time bus stop information (second entry from top, [example query](http://lv.raad.tartu.ee:8877/SmartBusSVC.asmx/GetBoardInfo?stop_id=7820321-1)). The example query does not return anything and the server is possibly firewalled. E-mailing a couple city contacts has yielded no answer whatsoever,  so I'm hoping that OK Estonia could use its clout here.

A related problem is that all Tartu city buses do have GPS receivers and submit this information to a central system, but these data do not seem to be provided even nominally (the API mentioned above should basically return information present on the electronic boards installed at bus stops, not actual coordinates of the buses). Of course it would be desirable to open up the GPS data as well.

The benefits are clear: we could have third-party real-time public transportation maps and more for Tartu, [as we have for Tallinn](https://gis.ee/tallinn/).

**Comment by boamaod** [comment]
I have tried to promote creating proper open data catalog for Tartu among city council factions and trying to make it an election topic, but my personal capacity is rather limited. There is city budget currently under discussion and I have suggested reserving resources for initialising open data portal (which would presumably entail some support for requests like you describe):

* https://www.ria.ee/ee/jooksev-voor-avaandmed-ja-andmeanaluus.html

This could be supported by plan to formulate our proposal on how open data should be managed at municipal level:

* https://github.com/okestonia/esindus/issues/23

If you want to help, you are very welcome, @tambeta!

**Comment by infokujur** [comment]
Andmed on avaldatud opendata.riik.ee lehel: https://opendata.riik.ee/andmehulgad/tartu-linna-uhistranspordi-peatuste-numbrid-nimekiri/

Kui andmetes on vaja parandust, siis palun avage uus issue.

**Comment by tambeta** [comment]
@infokujur, siin teemas on juttu reaalajaandmetest. Need ei ole vabalt kättesaadavad.
